libs/components/forms/src/lib/modules/input-box/input-box.component.scss

```diff
@@ -415,7 +464,7 @@ sky-input-box {
         /* Ensure the form control border shows over the adjacent input group button. */
         z-index: 1;
 
-        .sky-form-group {
+        .sky-form-group:not(:has(:autofill)):not(:has(:-webkit-autofill)) {
```

Comment:
We have to place the borders/box-shadows on the autofilled element or there is an odd double border that doesn't look right due to how autofill styles are applied. So we want to turn off the standard ones when autofill is applied.